### PR TITLE
20-10207 - Add front-end tests (clean)

### PR DESCRIPTION
**Pull request by tlei123**
## Summary

- Add front-end tests to Priority Processing (20-10207)
- Team: Veteran Facing Forms
- Flipper not implemented yet

## Related issue(s)

- department-of-veterans-affairs/va.gov-team-forms#1075

## Testing done

- Local unit- & e2e-tests all successful

## What areas of the site does it impact?

This form ONLY

### CODE REVIEW

File: src/applications/simple-forms/20-10207/tests/unit/components/AlsViewField.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  afterEach(done => {
+    if (wrapper) {
+      wrapper.unmount();
+      wrapper = null;
+    }
+    done();
+  });
+
+  it('renders without crashing', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/AlsViewField.unit.spec.jsx:26]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/AlsViewField.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    wrapper = mount(
+      <AlsViewField defaultEditButton={() => {}} formData={mockFormData} />,
+    );
+    expect(wrapper.exists()).to.equal(true);
+    done();
+  });
+
+  it('renders ALS_DESCRIPTION', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/AlsViewField.unit.spec.jsx:35]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/AlsViewField.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    wrapper = mount(
+      <AlsViewField defaultEditButton={() => {}} formData={mockFormData} />,
+    );
+    expect(wrapper.contains(ALS_DESCRIPTION)).to.equal(true);
+    done();
+  });
+
+  it('renders alsDocuments when provided', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/AlsViewField.unit.spec.jsx:44]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/FinancialHardshipViewField.unit.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  afterEach(done => {
+    if (wrapper) {
+      wrapper.unmount();
+      wrapper = null;
+    }
+    done();
+  });
+
+  it('renders without crashing', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/FinancialHardshipViewField.unit.spec.jsx:26]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/FinancialHardshipViewField.unit.spec.jsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        defaultEditButton={() => {}}
+        formData={mockFormData}
+      />,
+    );
+    expect(wrapper.exists()).to.equal(true);
+    done();
+  });
+
+  it('renders FINANCIAL_HARDSHIP_DESCRIPTION', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/FinancialHardshipViewField.unit.spec.jsx:38]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/FinancialHardshipViewField.unit.spec.jsx [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        defaultEditButton={() => {}}
+        formData={mockFormData}
+      />,
+    );
+    expect(wrapper.contains(FINANCIAL_HARDSHIP_DESCRIPTION)).to.equal(true);
+    done();
+  });
+
+  it('renders financialHardshipDocuments when provided', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/FinancialHardshipViewField.unit.spec.jsx:50]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/MedalAwardViewField.unit.spec.jsx [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  afterEach(done => {
+    if (wrapper) {
+      wrapper.unmount();
+      wrapper = null;
+    }
+    done();
+  });
+
+  it('renders without crashing', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/MedalAwardViewField.unit.spec.jsx:26]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/MedalAwardViewField.unit.spec.jsx [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+        defaultEditButton={() => {}}
+        formData={mockFormData}
+      />,
+    );
+    expect(wrapper.exists()).to.equal(true);
+    done();
+  });
+
+  it('renders MEDAL_AWARD_DESCRIPTION', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/MedalAwardViewField.unit.spec.jsx:38]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/MedalAwardViewField.unit.spec.jsx [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+        defaultEditButton={() => {}}
+        formData={mockFormData}
+      />,
+    );
+    expect(wrapper.contains(MEDAL_AWARD_DESCRIPTION)).to.equal(true);
+    done();
+  });
+
+  it('renders medalAwardDocuments when provided', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/MedalAwardViewField.unit.spec.jsx:50]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/PowViewField.unit.spec.jsx [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+  afterEach(done => {
+    if (wrapper) {
+      wrapper.unmount();
+      wrapper = null;
+    }
+    done();
+  });
+
+  it('renders without crashing', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/PowViewField.unit.spec.jsx:26]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/PowViewField.unit.spec.jsx [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+    wrapper = mount(
+      <PowViewField defaultEditButton={() => {}} formData={mockFormData} />,
+    );
+    expect(wrapper.exists()).to.equal(true);
+    done();
+  });
+
+  it('renders POW_DESCRIPTION', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/PowViewField.unit.spec.jsx:35]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/PowViewField.unit.spec.jsx [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+    wrapper = mount(
+      <PowViewField defaultEditButton={() => {}} formData={mockFormData} />,
+    );
+    expect(wrapper.contains(POW_DESCRIPTION)).to.equal(true);
+    done();
+  });
+
+  it('renders powDocuments when provided', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/PowViewField.unit.spec.jsx:44]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/TerminalIllnessViewField.unit.spec.jsx [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+  afterEach(done => {
+    if (wrapper) {
+      wrapper.unmount();
+      wrapper = null;
+    }
+    done();
+  });
+
+  it('renders without crashing', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/TerminalIllnessViewField.unit.spec.jsx:26]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/TerminalIllnessViewField.unit.spec.jsx [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+        defaultEditButton={() => {}}
+        formData={mockFormData}
+      />,
+    );
+    expect(wrapper.exists()).to.equal(true);
+    done();
+  });
+
+  it('renders TERMINAL_ILLNESS_DESCRIPTION', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/TerminalIllnessViewField.unit.spec.jsx:38]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/TerminalIllnessViewField.unit.spec.jsx [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+        defaultEditButton={() => {}}
+        formData={mockFormData}
+      />,
+    );
+    expect(wrapper.contains(TERMINAL_ILLNESS_DESCRIPTION)).to.equal(true);
+    done();
+  });
+
+  it('renders terminalIllnessDocuments when provided', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/TerminalIllnessViewField.unit.spec.jsx:50]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/VsiViewField.unit.spec.jsx [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
+    wrapper = mount(
+      <VsiViewField defaultEditButton={() => {}} formData={mockFormData} />,
+    );
+    expect(wrapper.exists()).to.equal(true);
+    done();
+  });
+
+  it('renders VSI_DESCRIPTION', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/VsiViewField.unit.spec.jsx:35]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

File: src/applications/simple-forms/20-10207/tests/unit/components/VsiViewField.unit.spec.jsx [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+    wrapper = mount(
+      <VsiViewField defaultEditButton={() => {}} formData={mockFormData} />,
+    );
+    expect(wrapper.contains(VSI_DESCRIPTION)).to.equal(true);
+    done();
+  });
+
+  it('renders vsiDocuments when provided', done => {
+    // eslint-disable-next-line @department-of-veterans-affairs/enzyme-unmount
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/simple-forms/20-10207/tests/unit/components/VsiViewField.unit.spec.jsx:44]: Yes, this is required.  For some reason, even though I'm unmounting wrapper in an afterEach block, eslint still flagged `mount` as having no unmount.

**Review by rhasselle-oddball — APPROVED** [review]
exemplary testing. looks great!

**Pull request merged**